Added additional information about managing gift registry items (#1472)
magento/merchdocs: src/customers/account-dashboard-gift-registry.md

1. Edit the required options, and click **Save**.

## Manage gift registry items

![]({% link images/images-ee/account-dashboard-gift-registry-items-management.png %}){: .zoom}
_Gift Registry items managing_

1. From customer account dashboard, go to **Gift Registry**.

> 1. Find the registry event, items which you want to manage, and click **Manage Items**.

Reviewer: ```suggestion
1. Find the registry event, then the items which you want to manage, and click **Manage Items**.
```

> 1. Change the needed options like: **Note**, **Qty** or **Delete** an item from gift registry by checking checkbox near needed item.

Reviewer: Suggest breaking up the line as it read a bit confused as there are two different types of action on the line.
```suggestion
1. Change the required options such as : **Note** &  **Qty** 

1.  Or **Delete** an item from the gift registry by checking the checkbox near the required item.
```

Author: Thank you for the review. Changes applied.
